Core/StarlightPlayer.Events.cs

```diff
@@ -64,7 +64,7 @@ public override void OnHitByProjectile(Projectile proj, Player.HurtInfo hurtInfo
 		public delegate void ModifyHitNPCDelegate(Player player, Item Item, NPC target, ref NPC.HitModifiers modifiers);
 		public override void ModifyHitNPCWithItem(Item item, NPC target, ref NPC.HitModifiers modifiers)
 		{
-			//AddHitPacket(null, target, damage, knockback, crit); PORTTODO: Change
+			AddHitPacket(null, target, modifiers);
```

Comment:
I wonder if it might be wise to have a seperate ModPlayer explicitly for sending these packets? How many of these are there?